purchase_security/models/purchase.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Tecnativa - Stefan Ungureanu
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    purchase_group_ids = fields.Many2one(
+        "res.groups",
```

Comment:
This should be a new model: `purchase.team`. And call this field `team_id` (without `s`), as it's a many2one and it's inside `purchase.order`, so `purchase` is redundant.